RevenueCatUI/CustomerCenter/ManageSubscriptionsButtonStyle.swift

```diff
@@ -66,7 +52,15 @@ struct ManageSubscriptionsButtonStyle_Previews: PreviewProvider {
     static var previews: some View {
         Button("Didn't receive purchase") {}
             .buttonStyle(ManageSubscriptionsButtonStyle())
-            .environment(\.appearance, CustomerCenterConfigData.Appearance(mode: .system))
+            .environment(\.appearance, CustomerCenterConfigData.Appearance(
```

Comment:
Nit: It'd be nice to extract this `CustomerCenterConfigData.Appearance` to a constant somewhere so that it can be reused across previews